Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift

```diff
@@ -4,13 +4,11 @@
 
 // Disabled: It will be updated in FXIOS-6128
 
-/*
 import Foundation
 import XCTest
 import SwiftUI
 @testable import Client
 
-
 class CreditCardInputFieldTests: XCTestCase {
```

Comment:
Same comment as above great coverage but it will be good to add test for edge cases